examples/set-labels-full-coverage/README.md

```diff
@@ -1,16 +1,16 @@
-# set-labels: Advanced Example
+# set-labels: Full Coverage Example
```

Comment:
What do we want to do in this example? Include all built-in resources?

Reply:
Yes, this covers all the resources in `CommonSpecs`